panels/class-debug-bar-deprecated.php

```diff
@@ -30,9 +30,11 @@ function prerender() {
 
 	function render() {
 		echo '<div id="debug-bar-deprecated">';
-		echo '<h2><span>', __( 'Total Functions:', 'debug-bar' ), '</span>', number_format_i18n( count( $this->deprecated_functions ) ), "</h2>\n";
-		echo '<h2><span>', __( 'Total Arguments:', 'debug-bar' ), '</span>', number_format_i18n( count( $this->deprecated_arguments ) ), "</h2>\n";
-		echo '<h2><span>', __( 'Total Files:', 'debug-bar' ), '</span>', number_format_i18n( count( $this->deprecated_files ) ), "</h2>\n";
+
+		$this->render_panel_info_block( __( 'Total Functions:', 'debug-bar' ), count( $this->deprecated_functions ) );
```

Comment:
Yes, the sanitization of the value was moved to the `render_panel_info_block()` method to be:
a) consistently done for all places this pattern is used
b) to be closer to the `echo` call doing the output.